Plugin/CppApi/include/CoordinateConversionController.h

```diff
@@ -13,15 +13,16 @@
 #ifndef COORDINATECONVERSIONCONTROLLER_H
 #define COORDINATECONVERSIONCONTROLLER_H
 
-#include <QQmlListProperty>
-
 #include "AbstractTool.h"
 #include "CoordinateConversionOptions.h"
 
 #include "SpatialReference.h"
 #include "Point.h"
 #include "GeometryTypes.h"
 
+#include <QQmlListProperty>
+#include <QAbstractListModel>
```

Comment:
Does `QAbstractListModel` need to be included here, or can the class just be forward declared?

Reply:
I think we need it since this is a property